lib/active_merchant/billing/gateways/datatrans.rb

```diff
@@ -72,6 +72,28 @@ def void(authorization, options = {})
         commit('cancel', post, { transaction_id: transaction_id })
       end
 
+      def store(payment_method, options = {})
+        exp_year = format(payment_method.year, :two_digits)
+        exp_month = format(payment_method.month, :two_digits)
+
+        post = {
+          requests: [
+            {
+              type: 'CARD',
+              pan: payment_method.number,
+              expiryMonth: exp_month,
+              expiryYear: exp_year
+            }
+          ]
+        }
+        commit('tokenize', post, { expiry_month: exp_month, expiry_year: exp_year })
+      end
+
+      def unstore(authorization, options = {})
+        data_alias = authorization.split('|')[2]
+        commit('delete_alias', {}, { alias_id: data_alias }, :delete)
+      end
+
```

Comment:
❓ Question:

Just asking should we scrub the alias ?

Reply:
I'm not sure, some gateways don't do it. Considering we are also passing expiration info. Filter the alias could be better.